test/unit/proxy_config_affecting_changes_test.rb

```diff
@@ -60,6 +60,38 @@ def self.column_names
     end
   end
 
+  class StoredModelTest < ActiveSupport::TestCase
+    class StoredModel < ActiveRecord::Base
+      include ProxyConfigAffectingChanges::ModelExtension
+      define_proxy_config_affecting_attributes :settings
+      self.table_name = 'gateway_configurations'
+
+      store :settings, accessors: [:jwt_claim_with_client_id, :jwt_claim_with_client_id_type], coder: JSON
```

Comment:
Not important, but codeclimate will probably suggest:

```suggestion
      store :settings, accessors: %i[jwt_claim_with_client_id jwt_claim_with_client_id_type], coder: JSON
```